discord_bot2.py

```diff
@@ -81,7 +82,7 @@ def __init__(self, prompt_dir: Path, conversation_dir: Path):
 
     def _load_prompts(self, prompt_dir: Path):
         self.prompts = {}
-        for file in prompt_dir.iterdir():
+        for file in prompt_dir.glob("**/*"):
```

Comment:
What if two files in different folders have the same name? 

I would keep the original behavior for now.